grails-app/assets/javascripts/asinha/customer-create-httpPOST.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function () {
+    $("form").on("submit", function(event) {
+        event.preventDefault();
+        let infosCustomer = {};
+        let data = new FormData(document.querySelector("form"));
+        data.forEach(function (value,key) {
+            infosCustomer[key] = value;
+        });
+        delete infosCustomer.cpfCnpjRadio;
+        $.post(document.querySelector("form").getAttribute("action"), infosCustomer, function(response) {
```

Comment:
para não ficar muito grande e difícil a leitura pode atribuir um valor da url numa variável
```suggestion
        var url = document.querySelector("form").getAttribute("action");
        $.post(url, infosCustomer, function(response) {
```